test/e2e/job.go

```diff
@@ -417,6 +417,106 @@ var _ = Describe("Job E2E Test", func() {
 		checkError(context, err)
 	})
 
+	FIt("Starvation prevention", func() {
+		context := initTestContext()
+		defer cleanupTestContext(context)
+		maxPods := clusterSize(context, oneCPU)
+
+		// create a job to prevent job "big" from running
+		replicaset := createReplicaSet(context, "rs1", maxPods-2, "nginx", oneCPU)
```

Comment:
Do you still plan to replace replicaset with job?

Reply:
it's replaced